build.gradle.kts

```diff
@@ -54,4 +54,6 @@ dependencies {
     testRuntimeOnly("org.junit.platform:junit-platform-suite-api:latest.release")
     testRuntimeOnly("ch.qos.logback:logback-classic:latest.release")
     testRuntimeOnly("com.squareup.okhttp3:mockwebserver:3.14.9")
+    testRuntimeOnly("org.powermock:powermock-api-mockito:1.6.5")
+    testRuntimeOnly("org.powermock:powermock-core:1.6.5")
```

Comment:
These can be moved to `classpathFromResources`
```suggestion
```

Reply:
Using the new mechanism isn't obligatory, except when one recipe needs to use multiple different versions of the same dependency. I don't believe that is the case here, so either mechanism should work fine

Reply:
I tried several things to get the powermock dependecy into the unit test. However it is still failing. Do you have any idea, what could be wrong?

Reply:
Fixed in https://github.com/openrewrite/rewrite-testing-frameworks/pull/304/commits/dea6eeb0470992978a441531d0e6c4d793b18967 as [discussed on Slack](https://rewriteoss.slack.com/archives/C01A843MWG5/p1675152813413409?thread_ts=1675097259.855829&cid=C01A843MWG5), with an [improvement logged](https://github.com/openrewrite/rewrite-docs/issues/124) for the documentation.